test/stringify.js

```diff
@@ -193,3 +193,32 @@ test('should disable sorting', t => {
 		sort: false
 	}), 'c=foo&b=bar&a=baz');
 });
+
+test('should ignore null when skipNullAndUndefined is set', t => {
+	t.is(queryString.stringify({
+		a: 1,
+		b: null,
+		c: 3
+	}, {
+		skipNullAndUndefined: true
+	}), 'a=1&c=3');
+});
+
+test('should ignore undefined when skipNullAndUndefined is set', t => {
+	t.is(queryString.stringify({
+		a: 1,
+		b: undefined,
+		c: 3
+	}, {
+		skipNullAndUndefined: true
+	}), 'a=1&c=3');
+});
+
+test('should ignore both null and undefined when skipNullAndUndefined is set', t => {
+	t.is(queryString.stringify({
+		a: undefined,
+		b: null
+	}, {
+		skipNullAndUndefined: true
+	}), '');
+});
```

Comment:
Add test cases for `arrayFormat`.

Reply:
Maybe also add test cases for when it's set to `false`? 🤔
'Cause I thought this was already the current behavior.